.circleci/config.yml

```diff
@@ -277,14 +277,6 @@ jobs:
           name: Run e2e tests
           command: yarn test:e2e:android
 
-  "Publish: semantic release":
-    <<: *js_defaults
-    steps:
-      - *addWorkspace
-      - run:
-          name: Publish to NPM registry
-          command: yarn ci:publish 
```

Comment:
If you remove this job, how is this going to be the last release? The previous one will be the last.

Reply:
I meant to release v1.5 manually